### DISCUSSION TITLE
[@types/react] missing HTML global attribute "part"

### DISCUSSION THREAD

**Discussion by sacru2red** [DefinitelyTyped/DefinitelyTyped#66186] [Issues with a @types package] [answered]
global attribute "part" is missing

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/part

I think we need to add part to namespace React > HTMLAttributes<T> prop

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
Both HTMLAttributes and SVGAttributes I assume. We can add this to all currently maintained versions. Contributions welcome.

**Comment by sacru2red** [comment]
#72219 #72230
